frontend/src/app/organization/organization-contact/organization-contact.component.html

```diff
@@ -12,34 +12,51 @@ <h2>Formularz kontaktowy</h2>
   </div>
   <form (ngSubmit)="onSubmit()" #contactForm="ngForm">
     <div class="form-group row">
-      <label for="id_name" class="col-md-3 col-form-label">Twoje imię i nazwisko</label>
-      <div class="col-md-9">
+      <label for="id_name" class="col-md-2 col-form-label">Twoje imię</label>
+      <div class="col-md-4">
         <input
           type="text"
           name="name"
           class="form-control form-control-lg"
           [ngClass]="{'is-invalid': name.invalid && name.touched}"
           id="id_name"
-          ngModel
+          [ngModel]="(user$ | async).firstName"
           required
           minlength="2"
           maxlength="150"
           #name="ngModel" />
         <div
           class="invalid-feedback"
-          *ngIf="name.invalid && name.touched">Proszę podać imię i nazwisko.</div>
+          *ngIf="name.invalid && name.touched">Proszę podać imię</div>
+      </div>
+      <label for="id_lastName" class="col-md-2 col-form-label">Twoje nazwisko</label>
```

Comment:
I know that there's a sentence regarding considering to split these two fields, but we have pretty similar form (to contact administrator) that is already prepopulated with user data and don't split first and last name. For coherency, can we leave it unsplitet here also?

For admin contact form see: #999 

You can even use function that was created there (`getFullName`): https://github.com/codeforpoznan/volontulo/blob/master/frontend/src/app/user.service.ts#L46